File: Frontend/watten-mobile/src/utils/Enums.ts
export enum RequestMethod {
  GET = "GET",
  POST = "POST",
  DELETE = "DELETE",
  PUT = "PUT",
}
export enum Mode {
  Add = 0,
  Edit = 1,
}

export enum Language {
  English = "en",
  Arabic = "ar",

  Hebrew = "he",
}

export enum UserRoles {
  Admin = 0,
  Instructor = 1,
  Parent = 2,
}

export enum SessionStatus {
  Available = 0,
  Full = 1,
  Finished = 2,
  Canceled = 3,
}

File: Backend/Infrastructure/Enums/Enums.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.Enums
{
    public class Enums
    {

        public enum SessionStatus
        {
            Available,
            Full,
            Started,
            Finished,
            Canceled
        }

        public enum Day
        {
            Sunady,
            Monday,
            Tuesday,
            Wednesday,
            Thursday,
            Friday,
            Saturday,
        }


        public enum UserRole
        {
            Admin,
            Instructor,
            Parent,
        }



        public enum Folder
        {
            Default,
            SessionImages,
            UserImages,
        }


        public enum NotificationType
        {
            Notification, Data, DataAndNotification
        }
        public enum NotificationSendType
        {
            Token, Topic
        }
    }
}
